requirements-dev.txt

```diff
@@ -2,6 +2,7 @@
 
 coverage
 flake8
+ipdb
```

Comment:
please don't add ipdb (rebase out) -- if you want this in your own environment install it there